doc/source/whatsnew/v0.18.0.txt

```diff
@@ -20,6 +20,21 @@ Check the :ref:`API Changes <whatsnew_0180.api>` and :ref:`deprecations <whatsne
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0180.enhancements.extractall:
+
+The ``.str.extractall`` method is added (:issue:`11386`).
+Unlike ``extract`` (which returns only the first match),
+
+.. ipython:: python
```

Comment:
so this is correct now, since we added a sub-section. If you didn't have a sub-section, then you would need to indent everything except the first line (e.g. including the ipython line)